src/main/java/frc/robot/auton/MovePercentOutput.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.auton;
+
+import frc.robot.QuickActions;
+
+public class MovePercentOutput extends AutonAction {
+
+    @Override
+    public boolean isDone() {
+        return true;
+    }
+
+    @Override
+    public void initiate() {
+        QuickActions.setDriveMotors(0.25);
+    }
+
+    @Override
+    public void shutdown() {}
+}
```

Comment:
This probably isn't needed anymore, this was a quick and dirty way to just move backwards in auton ,without relying on PID's.